editor/components/writing-flow/index.js

```diff
@@ -212,6 +252,32 @@ class WritingFlow extends Component {
 		}
 	}
 
+	componentDidUpdate( prevProps ) {
+		// When selecting a new block, we focus its first editable or the container
```

Comment:
This is pretty jarring for nested blocks when "selecting" occurs by opening "More Options". Looks like we somehow exempted a non-nested block from this behavior?

When trying to open More Options for a nested block which is not currently selected, it forces focus into the first column instead of opening the menu.

Reply:
Mmm! why it's not opening the menu even if the focus is moved to the first Column?

I'd expect the order of things to be:

 - Callback to select the block
 - This should rerender and focus the first column
 - Next step in the callback to open the menu (`setState`)
 - This should rerender and open the menu and moves the focus to the first element in the menu.

I guess the order of things is probably not this one and maybe we can tweak the callback function to match this?

Reply:
Since the nested block is a different block, it would cause the parent block to become unselected when it's focused.